### PR TITLE
Calendar component 

### PR DESCRIPTION
**Pull request by dmrt2002**
updated the calendar component with the theme and tested the server config

### CODE REVIEW

File: src/components/tuComponent/index.ts [hunk 1]
```diff
@@ -32,7 +32,7 @@ export default defineComponent({
 		};
 	},
 	props: {
-		color: { type: String, default: null },
```
Comment (TheCelebrimbor) [inline src/components/tuComponent/index.ts]: Why was this removed ?

File: src/components/tuTable/tuTable.vue [hunk 2]
```diff
@@ -34,8 +34,9 @@
 		>
 			<table class="tu-table__element" ref="tableElement">
 				<thead ref="thead" class="tu-table__thead">
+					<slot v-if="$slots.thead" name="thead" />
```
Comment (TheCelebrimbor) [inline src/components/tuTable/tuTable.vue]: Why are there changes in table for a Calendar component PR ?
If so, what are these changes for ?

File: tsconfig.json [hunk 3]
```diff
@@ -37,7 +37,7 @@
     "src/**/*.vue",
     "tests/**/*.ts",
     "tests/**/*.tsx"
-  ],
+, "src/components/tuCalendar/main.js"  ],
```
Comment (TheCelebrimbor) [inline tsconfig.json]: Why do we need to add this ?

File: src/components/TestComponent2.vue [hunk 4]
```diff
@@ -0,0 +1,136 @@
+<template>
```
Comment (TheCelebrimbor) [inline src/components/TestComponent2.vue]: Can we delete this file ?

File: src/components/index.ts [hunk 5]
```diff
@@ -24,9 +24,11 @@ export { Notification, NotificationAttributes } from "./tuNotifications";
 export { tuSidebar, tuSidebarGroup, tuSidebarItem } from "./tuSideBar";
 export { tuPopper, tuPopupMenu, tuPopupItem } from "./tuPopper";
 export { default as tuDialog } from "./tuDialog";
-
 export { tuCollapse, tuCollapseItem } from "./tuCollapse";
 export { tuTreeView, tuTreeItem } from "./tuTreeView";
+export { tuCalendar } from "./tuCalendar";
+export { tuColorPicker } from "./tuColorPicker";
+export { tuWysiwyg } from "./tuWysiwyg/index";
```
Comment (TheCelebrimbor) [inline src/components/index.ts]: export { tuWysiwyg } from "./tuWysiwyg" should be sufficient

**Review by rvndbalaji — APPROVED** [review]
LGTM - Please resolve conflicts

File: src/components/tuComponent/index.ts (hunk 1 above)
Comment (dmrt2002) [reply src/components/tuComponent/index.ts]: i was testing out and playing around the tucomponent to implement that toggle dark mode so will fix that

File: src/components/tuTable/tuTable.vue (hunk 2 above)
Comment (dmrt2002) [reply src/components/tuTable/tuTable.vue]: i was working on the slots thing in calendar so was refering the tutable component so while doing that i missed something will fix that

File: tsconfig.json (hunk 3 above)
Comment (dmrt2002) [reply tsconfig.json]: will remove that

File: src/components/TestComponent2.vue (hunk 4 above)
Comment (dmrt2002) [reply src/components/TestComponent2.vue]: okayy

File: src/components/index.ts (hunk 5 above)
Comment (dmrt2002) [reply src/components/index.ts]: yeah
